CAFAna/Analysis/Exposures.h

```diff
@@ -16,6 +18,33 @@ namespace ana
   //*======================================================================================================*/
 
   const double POT120 = 1.1e21;
-  const double POT80 = 1.47e21;
-  const double POT60 = 1.89e21;
+  const double POT80  = 1.47e21;
```

Comment:
These numbers are the energy of the proton beam, right? Can we add a comment saying so?